widgetssdk/build.gradle

```diff
@@ -7,8 +7,8 @@ android {
     defaultConfig {
         minSdkVersion 24
         targetSdkVersion 30
-        versionCode 21
-        versionName "1.6.16"
+        versionCode 22
+        versionName "1.6.17"
```

Comment:
there are 2 improvement tasks in prelive branch, and the version got bumped there as well. So if you want to release those improvements, pls merge prelive to master.

Reply:
i've dropped this commit and changed the base branch. so now it should look correct